flixel/FlxCamera.hx

```diff
@@ -835,7 +835,7 @@ class FlxCamera extends FlxBasic
 				position = renderPoint.set();
 			
 			drawVertices.length = 0;
-			final verticesLength = vertices.length;
+			final verticesLength = Std.int(vertices.length / 2) * 2;
```

Comment:
Do you know of a case where we would need to round to the nearest multiple? is it possible to add an incorrect multiple? if so we shouldn't allow that, assume it was a mistake and throw an error then, rather than accounting for it everywhere